plugins/node/opentelemetry-instrumentation-dns/src/instrumentation.ts

```diff
@@ -27,26 +27,24 @@ import { DnsInstrumentationConfig } from './types';
 import * as utils from './utils';
 import { VERSION } from './version';
 import {
-  Dns,
-  DnsPromises,
```

Comment:
these imports invalidated the `internal-types.ts` guarntee, as they were exported in the public API from an internal file